rules/qunit.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  plugins: [
+    'qunit',
+  ],
+  rules: {
+    'qunit/assert-args': 'error',
+    'qunit/literal-compare-order': 'error',
+    'qunit/no-arrow-tests': 'error',
+    'qunit/no-assert-equal': 'off',
+    'qunit/no-assert-logical-expression': 'off',
+    'qunit/no-async-in-loops': 'error',
+    'qunit/no-async-test': 'error',
+    'qunit/no-commented-tests': 'error',
+    'qunit/no-compare-relation-boolean': 'error',
+    'qunit/no-conditional-assertions': 'error',
+    'qunit/no-early-return': 'off',
+    'qunit/no-global-assertions': 'off',
+    'qunit/no-global-expect': 'off',
+    'qunit/no-global-module-test': 'off',
```

Comment:
Nope. The rule doesn't support destructuring `test` out of `QUnit` or importing it like `import { test } from 'qunit'`. I initially had it enabled but testing on some of our repos brought this to light. Same for the other `no-global` rules.